typescript/src/api.ts

```diff
@@ -455,7 +455,14 @@ export class OrchestrateApi {
     const headers = {
       "Content-Type": "text/plain",
     } as { [key: string]: string; };
-    const route = this.getIDDependentRoute("/convert/v1/hl7tofhirr4", request.patientID);
+    const parameters = new URLSearchParams();
+    if (request.patientID) {
+      parameters.append("patientID", request.patientID);
+    }
+    let route = "/convert/v1/hl7tofhirr4";
+    if (parameters.toString()) {
```

Comment:
minor: maybe `if (parameters.size) {` so that we don't have to call `toString()` twice?